### PR TITLE
Components

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Deleted the boilerplate files.
- Created the `components` folder with the `calculator.js` file exporting the `Calculator` component.
- Added styling to the web app in `index.css`.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+  }
+}
+
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <CalculatorApp />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```
Comment (Emmyn5600) [inline src/index.js]: - [x] Could you please split out App.js and index.js. 

App.js should be inside the src/App.js

`class CalculatorApp extends React.PureComponent {
  render() {
    return (
      <div className="container">
        <h1>The Calculator Web App</h1>
        <Calculator />
      </div>
    );
  }
}`

and index.js file should be like this.

`ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root'),`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @rpire 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Great design matching with the given design:heavy_check_mark:

Changes required :writing_hand: 

  - Kindly, check it again for the linters configurations, you are missing some files in the configuration. please fix it.
  
   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by rpire** [comment]
Hello, @Emmyn5600 👋

I have clear what needs to be done regarding the missing `App.js`, but I'm a little confused about the missing Linter files.

I am using the `React/Redux` Linter files provided by Microverse but I don't know the files that I'm missing or where to find them. Would you kindly tell me what files am I missing?

Thank you for reviewing my work, I will proceed to implement the requested changes and request another review. 🔍

Cheers and Happy coding! 🤓

File: src/index.js (hunk 1 above)
Comment (rpire) [reply src/index.js]: Requirement satisfied.

**Review by oxenprogrammer — APPROVED** [review]
Hi @rpire , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello @oxenprogrammer 👋

Thank you for reviewing my work! I will proceed to merge the changes into the `dev` branch. 🤓

Cheers and Happy coding. 🚀

**Pull request merged**